File: 5/recherche.c
#include <stdio.h>

int main()
{
        int nb = 21;
        int temp = 0;
        while (temp != 20)
        {
                for(int i = 1; i <= 20; i++)
                {
                        temp = i;
                        if (nb % i != 0)
                        {
                                break;
                        }
                }
                nb++;
        }
        printf("%d\n", nb - 1);
        return 0;
}

File: 5/recherche.py
def main():  #recherche trop long, algorithme à améliorer
    nb = 21
    temp = 0
    while temp != 20:
        for i in range(1, 21):
            temp = i
            if nb % i != 0:
                break
        nb += 1
    print(nb - 1)


main()
